concourse/pipelines/templates/perf_pipeline-tpl.yml

```
@@ -121,6 +122,18 @@ resources:
     tag: latest
     username: _json_key
     password: ((pxf-cloudbuild-service-account-key))
+{% endif %}
+
+{% if redhat_major_version == 8 %}
```

```suggestion
{% elif redhat_major_version == 8 %}
```

[Jinja Documentation - If](https://jinja.palletsprojects.com/en/3.0.x/templates/#if)